src/global_helpers.R

```diff
@@ -749,3 +750,55 @@ update_product_statuses <- function(network, domain){
 
     return()
 }
+
+unit_converter <- function(val, input_unit, output_unit) {
+    
+    units <- tibble(prefix = c('n', "u", "m", "c", "d", "h", "k", "M"),
+        convert_factor = c(0.000000001, 0.000001, 0.001, 0.01, 0.1, 100, 
+            1000, 1000000))
+
+    old_fraction <- as.vector(str_split_fixed(input_unit, "/", n = Inf))
```

Comment:
did you add str_split_fixed to function_aliases.R? if not, gotta prepend `package_name::`

Reply:
I did not, but I will. Seems like it keeps coming up.